app/reducers/index.js

```diff
@@ -31,31 +31,21 @@ const defaultState = {
       currentRating: 10,
       ratings: [],
     },
-    // {
-    //   id: 3,
-    //   name: 'Be cool',
-    //   created: 1488984810658,
-    //   avatar: 'pepper',
-    //   status: 0,
-    //   currentRating: 10,
-    //   ratings: [],
-    // },
-    // {
-    //   id: 4,
-    //   name: 'Be cool',
-    //   created: 1488984810658,
-    //   avatar: 'pepper',
-    //   status: 0,
-    //   currentRating: 10,
-    //   ratings: [],
-    // },
   ],
+  step: {
+    currentStep: 'GOALS_LIST',
+  },
 };
 
 export default (state = defaultState, action) => {
   switch(action.type) {
-    // cases here
+    case 'STEP_ADD_GOAL':
+      return Object.assign({}, state, {
+        step: {
+          currentStep: 'ADD_GOAL',
```

Comment:
Ok This string `ADD_GOAL` is repeated in the code, Maybe make a file called steps.js that acts the same way as `action_types.js` will